bump crengine: hyphenation, kerning and epub.css fixes

poire-z opened the pull request:
Includes:
- Freetype kerning: fix possible unstable rendering https://github.com/koreader/crengine/pull/291
- Hyphenation: fix one-letter patterns, update French.pattern https://github.com/koreader/crengine/pull/290 Thanks @cramoisi 
- Hyphenation: update French.pattern https://github.com/koreader/crengine/pull/292
- epub.css: add style for empty-line (used on text files) https://github.com/koreader/crengine/pull/293
    Closes #5042

Frenzie commented:
Looks like https://github.com/koreader/crengine/pull/291 may have caused some page shifting? Needs a manual double check in any case. :-)

```
[ RUN      ] spec/front/unit/readerrolling_spec.lua @ 200: Readerrolling module test initialization should emit PageUpdate event after book is rendered
spec/front/unit/readerrolling_spec.lua:204: Expected objects to be the same.
Passed in:
(number) 7
Expected:
(number) 6

stack traceback:
	spec/front/unit/readerrolling_spec.lua:204: in function 'handleEvent'
	frontend/ui/widget/container/widgetcontainer.lua:88: in function 'propagateEvent'
	frontend/ui/widget/container/widgetcontainer.lua:106: in function 'handleEvent'
	frontend/apps/reader/modules/readerrolling.lua:184: in function 'setupXpointer'
	frontend/apps/reader/modules/readerrolling.lua:254: in function 'handleEvent'
	frontend/ui/widget/container/widgetcontainer.lua:88: in function 'propagateEvent'
	...
	/home/ko/.luarocks/share/lua/5.1/busted/block.lua:155: in function 'execute'
	/home/ko/.luarocks/share/lua/5.1/busted/init.lua:7: in function 'executor'
	/home/ko/.luarocks/share/lua/5.1/busted/core.lua:312: in function </home/ko/.luarocks/share/lua/5.1/busted/core.lua:312>

[  FAILED  ] spec/front/unit/readerrolling_spec.lua @ 200: Readerrolling module test initialization should emit PageUpdate event after book is rendered (95.30 ms)
```

poire-z commented:
Or may be just a unit test on a text file :) that broke with #3996 where we switched to using epub.css on all formats... I guess I didn't see I was breaking text file, and assumed it was some other stuff, andjust fixed that unit test :|

poire-z commented:
In #3996:
>  fix unit test
sample.txt looks as fine with epub.css than with the obsoleted
txt.css (just a little more condensed, so a lower number of pages).

condensed, that was it :)
